examples/training_without_local_storage.ipynb

```diff
@@ -0,0 +1,917 @@
+    "\n",
+    "* [Object Store](https://docs.mosaicml.com/en/stable/api_reference/composer.utils.object_store.object_store.html#composer.utils.object_store.object_store.ObjectStore): The abstract [Object Store](https://docs.mosaicml.com/en/stable/api_reference/composer.utils.object_store.object_store.html#composer.utils.object_store.object_store.ObjectStore) class provides an API for uploading and downloading checkpoints. Composer includes object store implementations for [S3](https://docs.mosaicml.com/en/stable/api_reference/composer.utils.object_store.s3_object_store.html#composer.utils.object_store.s3_object_store.S3ObjectStore), [SFTP](https://docs.mosaicml.com/en/stable/api_reference/composer.utils.object_store.sftp_object_store.html#composer.utils.object_store.sftp_object_store.SFTPObjectStore), and [Libcloud](https://docs.mosaicml.com/en/stable/api_reference/composer.utils.object_store.libcloud_object_store.html#composer.utils.object_store.libcloud_object_store.LibcloudObjectStore). You can also write your own implementation by extending the [base class](https://docs.mosaicml.com/en/stable/api_reference/composer.utils.object_store.object_store.html#composer.utils.object_store.object_store.ObjectStore) if you are using a custom backend.\n",
+    "\n",
+    "\n",
+    "We'll use these components together to back up our checkpoints and Tensorboard TF Event files to the cloud. Internally, the [`CheckpointSaver`](https://docs.mosaicml.com/en/stable/api_reference/composer.callbacks.checkpoint_saver.html#composer.callbacks.checkpoint_saver.CheckpointSaver) callback and [`TensorboardLogger`](https://docs.mosaicml.com/en/stable/api_reference/composer.loggers.tensorboard_logger.html#composer.loggers.tensorboard_logger.TensorboardLogger) pass all generated files to the [`Logger`](https://docs.mosaicml.com/en/stable/api_reference/composer.loggers.logger.html#composer.loggers.logger.Logger) as artifacts by calling [`Logger.file_artifact`](https://docs.mosaicml.com/en/stable/api_reference/composer.loggers.logger.html#composer.loggers.logger.Logger.file_artifact). The Logger then passes these files to each [`LoggerDestination`](https://docs.mosaicml.com/en/stable/api_reference/composer.loggers.logger_destination.html#composer.loggers.logger_destination.LoggerDestination) (specified via the `loggers` argument of the [Trainer constructor](https://docs.mosaicml.com/en/stable/api_reference/composer.trainer.trainer.html#composer.trainer.trainer.Trainer). A [`LoggerDestination`](https://docs.mosaicml.com/en/stable/api_reference/composer.loggers.logger_destination.html#composer.loggers.logger_destination.LoggerDestination), which can implement the [`log_file_artifact`](https://docs.mosaicml.com/en/stable/api_reference/composer.loggers.logger_destination.html#composer.loggers.logger_destination.LoggerDestination.log_file_artifact) method, is responsible for uploading the file to the cloud.\n",
```

Comment:
```suggestion
    "We'll use these components together to back up our checkpoints and Tensorboard TF Event files to the cloud. Internally, the [`CheckpointSaver`](https://docs.mosaicml.com/en/stable/api_reference/composer.callbacks.checkpoint_saver.html#composer.callbacks.checkpoint_saver.CheckpointSaver) callback and [`TensorboardLogger`](https://docs.mosaicml.com/en/stable/api_reference/composer.loggers.tensorboard_logger.html#composer.loggers.tensorboard_logger.TensorboardLogger) pass all generated files to the [`Logger`](https://docs.mosaicml.com/en/stable/api_reference/composer.loggers.logger.html#composer.loggers.logger.Logger) as artifacts by calling [`Logger.file_artifact`](https://docs.mosaicml.com/en/stable/api_reference/composer.loggers.logger.html#composer.loggers.logger.Logger.file_artifact). The Logger then passes these files to each [`LoggerDestination`](https://docs.mosaicml.com/en/stable/api_reference/composer.loggers.logger_destination.html#composer.loggers.logger_destination.LoggerDestination) (specified via the `loggers` argument of the [Trainer constructor](https://docs.mosaicml.com/en/stable/api_reference/composer.trainer.trainer.html#composer.trainer.trainer.Trainer)). A [`LoggerDestination`](https://docs.mosaicml.com/en/stable/api_reference/composer.loggers.logger_destination.html#composer.loggers.logger_destination.LoggerDestination), which can implement the [`log_file_artifact`](https://docs.mosaicml.com/en/stable/api_reference/composer.loggers.logger_destination.html#composer.loggers.logger_destination.LoggerDestination.log_file_artifact) method, is responsible for uploading the file to the cloud.\n",
```

Reply:
Also would be nice to make the linking consistent (either adding `` to all or not adding it to any).

Reply:
Decided to remove all the tics because they did not format properly when embedded in sphinx